src/plugins/policyPack/index.ts

```diff
@@ -274,13 +275,69 @@ export default class PolicyPackPlugin extends Plugin {
     }
   }
 
+  private getLinkedData({ providerData }: { providerData: ProviderData }): any {
```

Comment:
maybe we can move this to somewhere else in the sdk and export it (it's just generically linking data)

Reply:
I agree. This could be reusable in the future, we can put it under the `utils` folder

Reply:
done!